README.md

```diff
@@ -142,10 +142,42 @@ You can create Windows installer only when running on Windows, the same is true
 #### App signing
 
 The Mac release supports [code signing](https://developer.apple.com/library/mac/documentation/Security/Conceptual/CodeSigningGuide/Procedures/Procedures.html). To sign the `.app` in the release image, include the certificate ID in the command as so,
-```
+```shell
 npm run release -- --sign A123456789
 ```
 
+#### Mac App Store
+You should install the Electron build for MAS
+```
+export npm_config_platform=mas
+rm -rf node_modules
+npm install
+```
+
+To sign your app for Mac App Store
+```shell
+npm run release -- --mas --mas-sign "3rd Party Mac Developer Application: Company Name (APPIDENTITY)" --mas-installer-sign "3rd Party Mac Developer Installer: Company Name (APPIDENTITY)"
+```
+
+Or edit the `app/package.json`, remove the `//` from `//codeSignIdentitiy` and update the values with your sign indentities
+```json
+  "//codeSignIdentitiy": {
```

Comment:
Yes, these are only the ID of the certificates, but users can pass that ids via command line too